tools/test-proxy/Azure.Sdk.Tools.TestProxy/README.md

```diff
@@ -22,7 +22,7 @@ There is a walkthrough through the process below in the [how do I use the test p
 2. Install test-proxy
 
 ```powershell
-> dotnet tool install azure.sdk.tools.testproxy --global --prerelease --add-source https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json
+> dotnet tool install azure.sdk.tools.testproxy --global --add-source https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json --version 1.0.0-dev.20211104.2
```

Comment:
For docs, do we want to pin the version or just pull whatever is latest, that way we don't have to update this readme all the time?

Reply:
@scbedd

Reply:
How do I change this ` --version 1.0.0-dev.20211104.2` to get the latest version?

Running the command without the version runs into this error.
```
C:\Users\sanallur\AppData\Local\Temp\tbcelt2a.qtr\restore.csproj : error NU1103: Unable to find a stable package azure.sdk.tools.testproxy with version (>= 0.0.0)
C:\Users\sanallur\AppData\Local\Temp\tbcelt2a.qtr\restore.csproj : error NU1103:   - Found 42 version(s) in https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json [ Nearest version: 1.0.0-dev.20211115.1 ]
C:\Users\sanallur\AppData\Local\Temp\tbcelt2a.qtr\restore.csproj : error NU1103:   - Found 0 version(s) in C:\Program Files\dotnet\sdk\NuGetFallbackFolder
```

And `--version latest` doesn't seem to work.

Reply:
For docs like this, I think `latest` (what this line was) is fine.

@benbp within the actual test code, Harsha is pulling the target docker tag from `eng/common/testproxy/docker-start-proxy.ps1`. This is just for documentation reasons.

Reply:
`--prerelease` flag wasn't working for me.

Reply:
@HarshaNalluru can you share command+error you were running? Works on my box at least. @scbedd yeah I think docker tag is right in the script, I just assume we don't want to the documentation to end up referencing a months old version later on.

Reply:
@benbp I actually tried this command locally, and it didn't work for me either. I could have sworn it absolutely did though, I wouldn't put an untested command into a readme.

Exact command:

`dotnet tool install azure.sdk.tools.testproxy --global --prerelease --add-source https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json`

Reply:
@benbp 
`dotnet tool install azure.sdk.tools.testproxy --global --prerelease --add-source https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json`

```
Unrecognized command or argument '--prerelease'
Usage: dotnet tool install [options] <PACKAGE_ID>

Arguments:
  <PACKAGE_ID>   The NuGet Package Id of the tool to install.
```

Reply:
So, instead of pointing to the specific version like `--version 1.0.0-dev.20211104.2`, if there is a tag that points to the latest release, that would be ideal for the readme.

Can someone help with that? 🙂

Reply:
Ok. I have a suggestion @HarshaNalluru

Reply:
```suggestion
> dotnet tool install azure.sdk.tools.testproxy --global --add-source https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json --version 1.0.0-dev*
```

Reply:
This will default latest. Thanks for following up on this!

Reply:
I like the wildcard approach as well. Also @HarshaNalluru it looks like `prerelease` was only added for dotnet 6 (https://github.com/dotnet/sdk/commit/b5cf72aaba8fb0a50dba3924faa8b873f731ede7), hence why you're not seeing it.